docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/CrossValidationSplit.cs

```diff
@@ -26,7 +25,9 @@ public static void Example()
             // So below, we specify Group column as the column containing the sampling keys.
             // If we pass that column to cross validation it would be used to break data into certain chunks.
             var folds = mlContext.Data.CrossValidationSplit(dataview, numberOfFolds: 3, samplingKeyColumnName: "Group");
-            PrintPreviewRows(folds[0]);
+            var trainSet = mlContext.Data.CreateEnumerable<DataPoint>(folds[0].TrainSet, reuseRowObject: false);
+            var testSet = mlContext.Data.CreateEnumerable<DataPoint>(folds[0].TestSet, reuseRowObject: false);
```

Comment:
for CrossValidationSplit we only need one dataset. we just need to cv-split it and show the fold. no need for a separate train and test. in fact cv is used when you don't have a test set, so it's better to remove it.

Reply:
Spoke offline and resolved. This is good as it is.